hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -237,6 +237,9 @@ spec:
   #         clusterDefaultNodeSelector: <node-selectors-labels>
   #         namespace1: <node-selectors-labels>
   #         namespace2: <node-selectors-labels>
+  #   auditConfig:
+  #     auditPolicy:
+  #       name: auditpolicy
```

Comment:
It's not clear without further documentation that `name` references a `ConfigMap`. Can we rename this field to `configMapName` or `configMapRef: {name: <name>}`?